config/Makefile.mingw64

```diff
@@ -156,7 +156,7 @@ MKEXE_BOOT=$(CC) $(CFLAGS) $(LDFLAGS) $(OUTPUTEXE)$(1) $(2)
 MKEXE_ANSI=$(FLEXLINK) -exe
 
 ### How to build a static library
-MKLIB=rm -f $(1); $(TOOLPREF)ar rc $(1) $(2); $(RANLIB) $(1)
+MKLIB=rm -f $(1) && $(TOOLPREF)ar rcs $(1) $(2)
```

Comment:
These two have been switched from using `RANLIB` to using the 's' option of ar, but the configure script still calls ranlib expressly.

Reply:
As mentioned in my commit message, Mingw uses the GNU version of `ar` for which I'm certain the `s` flag is supported.  In contrast the `ar` provided by old Unix platforms could lack the `s` flag and could require an explicit `ranlib` call.

Reply:
Sorry, I missed the commit message thanks to GitHub's tedious interface!

What bothers me with this is twofold - a difference here is "annoying" (or at least a special case) when the 4 Windows ports get moved into the configure script with autoconf. It's also a bit weird that both `RANLIB` and `ar -s` are used at various points in the build system, though I don't expect that can ever cause a problem (I'd be more inclined to have altered the `#ml` branch to be calling `RANLIBCMD` and add the optimisation to use GNU ar's `-s` switch everywhere properly at a later date)